pkg/restapi/vc/operation/operations_test.go

```diff
@@ -1551,7 +1551,7 @@ func TestIssueCredential(t *testing.T) {
 
 		op.didBlocClient = &didbloc.Client{CreateDIDValue: didDoc}
 
-		issueCredentialHandler := getHandler(t, op, issueCredentialPath, mode)
+		issueCredentialHandler := getHandler(t, op, issueCredentialPath, issuerMode)
```

Comment:
It makes more sense to change name of issuerCredentialHandler to credentialHandler. Edge service operates in two modes: "issuer" and "verifier" so issuerMode doesn't make sense.

Reply:
it's "issueCredentialHandler" (name based on issue credential API) not "issuerCredentialHandler".